src/main/java/taxi/web/filter/RoleFilter.java

```diff
@@ -0,0 +1,52 @@
+package taxi.web.filter;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class RoleFilter implements Filter {
```

Comment:
are you sure that you need role for this program. In our task we have only 1 role is driver as user that`s all

Reply:
this is own wish to set role filter and registration controller